controllers/Like.controller.js

```diff
@@ -0,0 +1,59 @@
+const userDb = require("../models/user.model");
+
+module.exports.getAllLikedVideos = async (req, res) => {
+  const { userId } = req.params;
+  try {
+    const user = await userDb.findById(userId);
+    const data = await user.execPopulate("likedVideos");
+    return res.status(200).json({ success: true, data: [...data.likedVideos] });
+  } catch (error) {
+    return res
+      .status(500)
+      .json({ success: false, message: "internal server error" });
+  }
+};
+
+module.exports.addToLikedVideos = async (req, res) => {
```

Comment:
You can group these exports together,  `module.exports = {getLikedVideos,addToLikedVideos.....}`